portfolio/src/main/webapp/index.html

```diff
@@ -59,8 +59,12 @@ <h2>Hidden Talent? </h2>
       <img src="/myImages/tournament.jpeg" alt="taekwondo" class="tournament"><br>
 
       <p>Click here to get a random quote from my favorite TV shows or fact about me:</p>
-      <button onclick="addRandomGreeting()">Click</button>
-      <div id="greeting-container"></div>
+      <button onclick="addRandomFacts()">Click</button>
+      <div id="container"></div><br>
```

Comment:
nit: it is better to use <br/>
Also better place one tag per line.

Reply:
what he means is `<br>` is not valid b/c it's not closed. You need to use `<br />` which is short for `<br></br>`

Reply:
Sounds good, I'll correct it, thanks